make/clusterctl.mk

```diff
@@ -1,13 +1,22 @@
 # Copyright 2023 D2iQ, Inc. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
+define AWS_CREDENTIALS
+[default]
+aws_access_key_id = $(AWS_ACCESS_KEY_ID)
+aws_secret_access_key = $(AWS_SECRET_ACCESS_KEY)
+aws_session_token = $(AWS_SESSION_TOKEN)
+endef
+
+AWS_B64ENCODED_CREDENTIALS ?= $(shell echo "$$AWS_CREDENTIALS" | base64 -w0)
+
 .PHONY: clusterctl.init
 clusterctl.init:
 	env CLUSTER_TOPOLOGY=true \
 			EXP_RUNTIME_SDK=true \
 			EXP_CLUSTER_RESOURCE_SET=true \
 			EXP_MACHINE_POOL=true \
-			AWS_B64ENCODED_CREDENTIALS= \
+			AWS_B64ENCODED_CREDENTIALS=$(AWS_B64ENCODED_CREDENTIALS) \
```

Comment:
I'd rather not do this if possible and use identities via the kubernetes API. Wdyt?

Reply:
That makes sense https://cluster-api-aws.sigs.k8s.io/topics/multitenancy.html#awsclusterstaticidentity 
Maybe I should just document how to create the necessary `Secret`? Do we also want to add an `identityRef` and `AWSClusterControllerIdentity ` to the ClusterClass template?

Reply:
Yeah that would be great. I remember there is a multi tenant cluster class released by capa (artifact attached to GitHub release). Could you look and see if that is a better base to pull in?

Reply:
👍, that one doesn't help it uses `AWSClusterRoleIdentity`, I'll add it as `kustomize` patch.

Reply:
Sounds good, thanks!